src/probnum/quad/__init__.py

```diff
@@ -6,28 +6,29 @@
 choosing points to evaluate the integrand based on said model.
 """
 
+from probnum.quad.solvers.policies import Policy, RandomPolicy
+from probnum.quad.solvers.stopping_criteria import (
+    BQStoppingCriterion,
+    IntegralVarianceToleranceStoppingCriterion,
+    MaxNevalsStoppingCriterion,
+    RelativeMeanChangeStoppingCriterion,
+)
+
 from ._bayesquad import bayesquad, bayesquad_from_data
 from ._integration_measures import GaussianMeasure, IntegrationMeasure, LebesgueMeasure
-from .bq_methods import (
-    BayesianQuadrature,
-    BQBeliefUpdate,
-    BQInfo,
-    BQStandardBeliefUpdate,
-    BQState,
-)
 from .kernel_embeddings import (
     KernelEmbedding,
     _kernel_mean_expquad_gauss,
     _kernel_mean_expquad_lebesgue,
     _kernel_variance_expquad_gauss,
     _kernel_variance_expquad_lebesgue,
 )
-from .policies import Policy, RandomPolicy
-from .stop_criteria import (
-    IntegralVarianceTolerance,
-    MaxNevals,
-    RelativeMeanChange,
-    StoppingCriterion,
+from .solvers import (
+    BayesianQuadrature,
+    BQBeliefUpdate,
```

Comment:
I would suggest moving components like `BQBeliefUpdate` and `BQStandardBeliefUpdate` under `quad.belief_updates`

Reply:
yes, but not in this PR. There are a couple of more things to address in the BQ package (see list at the end of #420 ). One of them is the directory structure, another one is the API reference structure (what the user sees). This PR simply enables the existing stopping criteria from the new base class, and so I moved the already to the correct directory.

Reply:
ah, sorry, I just saw your general comment now. I'd like to fix the API reference structure once the code structural changes are somewhat addressed, and the directory structure is more solid. I may ping you for this again, so it's in line with the changes in the linalg package.

Reply:
#574